### PR TITLE
template html cleanup

### PR DESCRIPTION
**Pull request by brendansudol**
consistent indention (2 spaces) - there was a mix of 4, 2, and misc spacings before

### CODE REVIEW

**Comment by mgwalker** [comment]
Let me go look at these 5,217 individual changes individually...  😜

**Comment by brendansudol** [comment]
fixing this conflict now...

File: DOL.WHD.Section14c.Web/src/modules/components/accountManagementControls/accountFormTemplate.html [hunk 1]
```diff
@@ -1,48 +1,65 @@
...
-                <li ng-repeat="error in savingErrors">{{error}}</li>
-            </u>
-        </p>
-    </div>
+  <div class="usa-alert-body">
+    <h3 class="usa-alert-heading">Error</h3>
+    <p class="usa-alert-text">An error occurred while saving the account.
+      <ul ng-show="savingErrors.length > 0">
+        <li ng-repeat="error in savingErrors">{{error}}</li>
+      </ul>
```
Comment (mgwalker) [inline DOL.WHD.Section14c.Web/src/modules/components/accountManagementControls/accountFormTemplate.html]: One of the things I like about React/JSX is that it'll catch unclosed or mismatched tags for you.  Maybe we should add an HTML linter as part of the tests.  🤔

**Comment by brendansudol** [comment]
make that 2,648 changes ;)

**Comment by brendansudol** [comment]
this is the last PR i'll do that touches so many things, if that's any consolation

**Pull request merged**